package-glue-job.sh

```diff
@@ -47,45 +47,37 @@ then
     pipenv requirements > requirements.txt
 
     echo "Installing dependencies..."
-    pip install -q -r requirements.txt --no-deps -t .
+    python -m pip install -q -r requirements.txt --no-deps -t .
 else
     echo "No Pipfile found, assuming no external dependencies are being used."
 fi
 
 # Create Wheel Package
 
-echo "* Creting setup.py file"
-cat >setup.py <<EOL
```

Comment:
I changed it so the script doesn't have to be first written to file, and then be filtered out by the code in the file.